public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -157,6 +159,12 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
   }
 
   query(request: DataQueryRequest<InfluxQuery>): Observable<DataQueryResponse> {
+    if (!this.isProxyAccess) {
+      const error = new Error(
+        'Browser access mode in the InfluxDB datasource is no longer available. Switch to server access mode.'
```

Comment:
We use this same string in 3 places, should we add a const and export this string so we're not redefining it in 3 files?